### ISSUE TITLE
CSS Filters: missing alt for animated images

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#330]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/css3filters/

`<img src="https://edgeportal.blob.core.windows.net/media/microsoft_loves_linux.jpg" class="img-responsive animated-filters">` 
and animated-filters2 image.

In this case the images are the result of the filter, so the alt text should be descriptive, saying what filter/animation is being applied, such as a photo of Satya with an opacity filter animated from 100% to 0% for the second one.
